tempesta_fw/http.h

```diff
@@ -614,6 +618,33 @@ tfw_http_resp_code_range(const int n)
 	return n <= HTTP_CODE_MAX && n >= HTTP_CODE_MIN;
 }
 
+/*
+ * Static index determination for response ':status' pseudo-header (see RFC
+ * 7541 Appendix A for details).
+ */
+static inline unsigned short
+tfw_h2_pseudo_index(unsigned short status)
+{
+	switch (status) {
```

Comment:
Wouldn't this be faster wit array?
Of course will use minimum 512 bytes.

Reply:
The compiler will do binary search over the labels which is much faster than accessing 4 cache line array. The compiler also generates lookup table (exactly the array you proposed), but for more localized values.